docs-content/release-notes.html.md.erb

```diff
@@ -34,7 +68,11 @@ Features in this release:
   could consume credentials for other services, if their field names exactly matched those used by 
   Conjur (e.g. version is a very common field).
 
-Known issues in this release: No known issues.
+Known issues in this release:
```

Comment:
Glad you added this retroactively!

Reply:
I actually just learned that this is recommended from here: https://github.com/pivotal-cf/docs-partners-template/blob/master/docs-content/release-notes.html.md.erb!!